docs/readmes/orc8r/development/minikube_deployment.md

```diff
@@ -80,7 +80,7 @@ $ cp -r ../../../../.cache/test_certs/* charts/secrets/.secrets/certs/.
 Create the kubernetes secrets
 
 ```
-helm template orc8r-secrets charts/secrets \
+helm template orc8r charts/secrets \
```

Comment:
This can conflict with the Orchestrator helm deployment having the same name: [helm install orc8r](https://github.com/magma/magma/blob/v1.4/docs/readmes/orc8r/development/minikube_deployment.md#install-with-helm)

Reply:
I think `helm template` is not creating the helm release, then it's okay.